src/main/scala/amailp/intellij/robot/psi/reference/MethodFinderWithoutUnderscoresAndSpaces.scala

```diff
@@ -0,0 +1,43 @@
+package amailp.intellij.robot.psi.reference
+
+import com.intellij.util.Processor
+import com.jetbrains.python.psi.PyElement
+
+/**
+ * @author Sergey Khomutinin skhomuti@gmail.com 
+ *         Date: 30.08.2019
+ */
+class MethodFinderWithoutUnderscoresAndSpaces(reference: String) extends Processor[PyElement] {
```

Comment:
Why do you extend here `Processor[PyElement]`?

I would rather extend `Processor[PyFunction]` (this is what `visitMethods` expects) and remove all the casts in the other file.